### PR TITLE
Use better variable names and correct variable initializations.

### PR DESCRIPTION
**Pull request by humblec**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by humblec** [comment]
@liggitt I know you are very busy on PRs, review ..etc. still managing to get all these done :)
 so no hurry, but would like to keep it in your radar!!

File: pkg/apis/core/validation/validation.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ const csiDriverNameRexpErrMsg string = "must consist of alphanumeric characters,
 const csiDriverNameRexpFmt string = `^[a-zA-Z0-9][-a-zA-Z0-9_.]{0,61}[a-zA-Z-0-9]$`
 
 var pdPartitionErrorMsg string = validation.InclusiveRangeError(1, 255)
-var fileModeErrorMsg string = "must be a number between 0 and 0777 (octal), both inclusive"
+var fileModeErrorMsg = "must be a number between 0 and 0777 (octal), both inclusive"
```
Comment (liggitt) [inline pkg/apis/core/validation/validation.go]: is there a reason to drop this type declaration and not the others around it?

Comment (humblec) [reply pkg/apis/core/validation/validation.go]: @liggitt linter give warning only on that declaration , may be its due to the presence of `"` in right hand side which explicitly make sure its a string. no warning I could see on other declarations.

Comment (liggitt) [reply pkg/apis/core/validation/validation.go]: I don't see a lot of value in sweeping to satisfy random golint complaints that don't make a lot of sense. @dims, can we take a step back on https://github.com/kubernetes/kubernetes/issues/68026 and figure out if we can scope the golint warnings and things we're fixing down to issues that are actually impactful (things like missing/nonstandard docs, etc)

**Pull request closed without merging**